apps/o2-blocks/index.php

```diff
@@ -63,7 +63,7 @@ function a8c_view_assets() {
 	if ( ! is_admin() ) {
 		$assets = require plugin_dir_path( __FILE__ ) . 'dist/view.asset.php';
 
-		$style_file = 'dist/view' . is_rtl() ? '.rtl.css' : '.css';
+		$style_file = 'dist/view' . ( is_rtl() ? '.rtl.css' : '.css' );
```

Comment:
Oh, sorry I missed that :( ... reminder to always test with a non-admin user or logged-out!

Reply:
Ah, `is_admin()` just checks if it's a wp-admin screen. So this would show up when viewing the blocks on the front-end

Reply:
Ah, true! I wonder why my test worked then 🤔

Reply:
Ah, I guess that was because I used the "Preview" instead of publishing and seeing the post in the actual frontend.

Reply:
oh interesting! I was also wondering the same thing.